app/workers/notifications/remove_all_worker.rb

```diff
@@ -0,0 +1,13 @@
+module Notifications
+  class RemoveAllWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(notifiable_ids, notifiable_type, service = Notifications::RemoveAll)
```

Comment:
Since we are not using the 3rd argument, `service = Notifications::RemoveAll`, can you remove it and just call it below? 

```
Notifications::RemoveAll.call(Array.wrap(notifiable_ids), notifiable_type)
```

Reply:
Of course. Sorry, I didn't remember to remove it in the args of the method.

Reply:
@mstruve I'm having a problem with this change. It seems in the specs the call isn't working and causing failures.

Reply:
You are going to have to update the specs to match with the new argument configuration for the worker. The worker expects 2 arguments which means the specs need to be updated to only send the worker 2 arguments.